pandas/tests/series/methods/test_truncate.py

```diff
@@ -141,3 +141,11 @@ def test_truncate_multiindex(self):
         expected = df.col
 
         tm.assert_series_equal(result, expected)
+
+    def test_truncate_one_element_series(self):
+        # GH 35544
+        series = pd.Series([0.1], index=pd.DatetimeIndex(["2020-08-04"]))
+        before = pd.Timestamp("2020-08-02")
+        after = pd.Timestamp("2020-08-04")
+
+        assert series.truncate(before=before, after=after).equals(series)
```

Comment:
can you use 

```
result = 
expected = 
tm.assert_series_equal(result, expected)
```

Reply:
I updated this one too. The input and expected series are the same. I  added a comment to clarify this.